pyvista/examples/examples.py

```diff
@@ -412,3 +412,95 @@ def load_nut():
 
     """
     return pyvista.read(os.path.join(dir_path, 'nut.ply'))
+
+
+def load_hydrogen_orbital(n=1, l=0, m=0, zoom_fac=1.0):
+    """Load the hydrogen wave function for a :class:`pyvista.UniformGrid`.
+
+    This is the Schrödinger equation evaluated in three dimensions.
+
+    Inspired by `Hydrogen Wave Function
+    <http://staff.ustc.edu.cn/~zqj/posts/Hydrogen-Wavefunction/>`_.
+
+    Parameters
+    ----------
+    n : int, default: 1
+        Principal quantum number. Must be a positive integer. This is often
+        referred to the "energy level" or "shell".
+
+    l : int, default: 0
+        Azimuthal quantum number. Must be a non-negative integer strictly
+        smaller than ``n``. By convention this value is represented by the
+        letters s, p, d, f, etc.
+
+    m : int, default: 0
+        Magnetic quantum number. Must be an integer ranging from ``-l`` to
+        ``l`` (inclusive). This is the orientation of the angular momentum in
+        space.
+
+    zoom_fac : float, default: 1.0
+        Zoom factor for the electron cloud. Increase this value to focus on the
+        center of the electron cloud.
+
+    Returns
+    -------
+    pyvista.UniformGrid
+        UniformGrid containing two ``point_data`` arrays:
+
+        * ``'real_hwf'`` - Real component of the quantum hydrogen wave function.
+        * ``'hwf'`` - Complex quantum hydrogen wave function.
```

Comment:
If it's not too much of a request:
```suggestion
        * ``'real_wf'`` - Real part of the wave function.
        * ``'wf'`` - Complex wave function.
```
I'd drop both "quantum" and "hydrogen", both are strongly implied. I'd also drop the "h" from the names. Might even consider renaming `real_wf` to `wf_real`? Not sure.